### PR TITLE
n0-computer.sendme version 0.3.0

### PR DESCRIPTION
**Pull request by legowerewolf**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/145988)

### CODE REVIEW

**Comment by legowerewolf** [comment]
@microsoft-github-policy-service agree

**Comment by legowerewolf** [comment]
I can't do anything to fix this. I suspect this is Windows Defender blocking it? It falsely detects a Trojan when I installed it locally, but when I scan the file itself, Defender clears it, and [nothing on VirusTotal picks up anything.](https://www.virustotal.com/gui/file/ffcac7eb9622bc9828cff90a7a655885e72c9c2fd7a374770846e30c93ae0fd4)

**Comment by stephengillie** [comment]
Hi @legowerewolf,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 870.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335187
> 2024-03-22 07:35:42.478 [FAIL] Installer failed security check. Url: https://github.com/n0-computer/sendme/releases/download/v0.3.0/sendme-v0.3.0-windows-x86_64.zip Result: 0x80004005

(Automated response - build 870.)

**Comment by legowerewolf** [comment]
Lol, that's what I thought.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\n0-computer.sendme__DefaultSource\sendme.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\sendme.exe returned exit code: -1073741515

(Automated response - build 873.)

File: manifests/n/n0-computer/sendme/0.3.0/n0-computer.sendme.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: n0-computer.sendme
+PackageVersion: 0.3.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: sendme.exe
+  PortableCommandAlias: sendme
+Installers:
```
Comment (stephengillie) [inline manifests/n/n0-computer/sendme/0.3.0/n0-computer.sendme.installer.yaml:11]:
```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 874.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**